### PR TITLE
Add host metrics page

### PR DESCRIPTION
**Pull request by thijsc**

### CODE REVIEW

File: source/getting-started/host-metrics.html.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+If you configure AppSignal with environment variables add this:
+
+`export APPSIGNAL_ENABLE_HOST_METRICS=true`
+
+The collected host metrics are available in the "Hosts" section you can
+reach through the left navigation in an app. Select a host and then
+click the "Metrics" tab to see all collected metrics.
+
+If you have any feedback on this new feature we'd love to hear it!
```
Comment (matsimitsu) [inline source/getting-started/host-metrics.html.md]: maybe add email to contact/support?

**Comment by matsimitsu** [comment]
👍

File: source/getting-started/host-metrics.html.md (hunk 1 above)
Comment (thijsc) [inline source/getting-started/host-metrics.html.md]: There's a contact us element almost right below it, so I thought that would be enough.

**Comment by matsimitsu** [comment]
:>

**Comment by wesoudshoorn** [comment]
I tweaked some minor copy stuff and added some layout. 👍

**Pull request merged**